--- idl_gen_rust_orig.cpp	2023-08-08 20:18:59
+++ idl_gen_rust.cpp	2023-08-08 20:21:13
@@ -406,7 +406,7 @@
     // example: f(A, D::E)          -> super::D::E
     // does not include leaf object (typically a struct type).
 
-    size_t i = 0;
+    // size_t i = 0;
     std::stringstream stream;
 
     auto s = src->components.begin();
@@ -417,7 +417,7 @@
       if (*s != *d) { break; }
       ++s;
       ++d;
-      ++i;
+      // ++i;
     }
 
     for (; s != src->components.end(); ++s) { stream << "super::"; }
